go.mod

```diff
@@ -34,7 +33,7 @@ require (
 	github.com/confluentinc/ccloud-sdk-go-v2/networking-privatelink v0.2.0
 	github.com/confluentinc/ccloud-sdk-go-v2/org v0.8.0
 	github.com/confluentinc/ccloud-sdk-go-v2/service-quota v0.2.0
-	github.com/confluentinc/ccloud-sdk-go-v2/srcm v0.5.0
+	github.com/confluentinc/ccloud-sdk-go-v2/srcm v0.5.1-0.20240130201821-ff257f548b9f
```

Comment:
Can we add a new version tag instead of using a commit?

Reply:
This is temporary until the PR is merged: https://github.com/confluentinc/ccloud-sdk-go-v2/pull/142